packages/account/src/Helpers/utils.tsx

```diff
@@ -181,12 +181,7 @@ export const isDocumentNumberValid = (document_number: string, document_type: Fo
                 document_name = 'document number';
                 break;
         }
-        return (
-            <Localize
-                i18n_default_text='Please enter your {{document_name}}. {{example_format}}'
-                values={{ document_name, example_format }}
-            />
-        );
+        return localize('Please enter your {{document_name}}. {{example_format}}', { document_name, example_format });
```

Comment:
just to confirm, will this work here? I mean passing the string and object in second parameter

Reply:
`example_format` came from a function where value is localized. `document_name` needed to be translated. Thanks for pointing it out